testing/unittests/jlibtests.cpp

```diff
@@ -85,6 +88,63 @@ class JlibSemTest : public CppUnit::TestFixture
 CPPUNIT_TEST_SUITE_REGISTRATION( JlibSemTest );
 CPPUNIT_TEST_SUITE_NAMED_REGISTRATION( JlibSemTest, "JlibSemTest" );
 
+
+class JlibSemTestStress : public CppUnit::TestFixture
+{
+public:
+    CPPUNIT_TEST_SUITE(JlibSemTestStress);
+        CPPUNIT_TEST(testSetup);
+        CPPUNIT_TEST(testSimple);
+        CPPUNIT_TEST(testCleanup);
+    CPPUNIT_TEST_SUITE_END();
+
+protected:
+
+    void testSetup()
+    {
+    }
+
+    void testCleanup()
+    {
+    }
+
+    void testTimedElapsed(Semaphore & sem, unsigned time, unsigned loopCount)
+    {
+        unsigned long sumTaken = 0;
```

Comment:
minor: if you want a 8 byte integer, we use a unsigned __int64, which guarantees it,